src/osd/ECTransaction.cc

```diff
@@ -134,6 +137,10 @@ struct TransGenerator : public boost::static_visitor<void> {
     assert(offset % sinfo.get_stripe_width() == 0);
     map<int, bufferlist> buffers;
 
+    bufferlist hbl;
+    op.oid.encode(hbl);
+    extent_cache.put(hbl.c_str(), offset, bl);
```

Comment:
cache addition should be in check_op() on-apply-all-done, before ackng client(TBD)
add/update cache perf counters(TBD)
cache usage to be restricted under feature fencing flag(TBD)